packages/@tailwindcss-upgrade/src/index.test.ts

```diff
@@ -149,6 +184,39 @@ it('should migrate a stylesheet (with imports)', async () => {
     @import './my-base.css' layer(base);
     @import './my-components.css' layer(components);
     @import './my-utilities.css' layer(utilities);
+
+    /*
+      The default border color has changed to \`currentColor\` in Tailwind CSS v4,
+      so we've added these compatibility styles to make sure everything still
+      looks the same as it did with Tailwind CSS v3.
+
+      If we ever want to remove these styles, we need to add an explicit border
+      color utility to any element that depends on these defaults.
+    */
+    @layer base {
+      *,
+      ::after,
+      ::before,
+      ::backdrop,
+      ::file-selector-button {
+        border-color: var(--color-gray-200, currentColor);
+      }
+    }
+    /*
```

Comment:
```suggestion
    }

    /*
```

Seems like something screws up the empty line between the two blocks 😢

Reply:
Figured it out. In this case the test is performing migrations and once done formats the nodes on a stringified version. The actual integration tests don't have this issue.

Reply:
If it works in the integration tests this is fine!